llvm/test/CodeGen/StubV/and.ll

```diff
@@ -0,0 +1,16 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 5
+; RUN: llc -mtriple=stubv32 < %s | FileCheck %s
+
+target datalayout = "e-m:e-p:32:32-i64:64-n32-S128"
+target triple = "stubv32"
+
+define i32 @and(i32 %a, i32 %b) {
+; CHECK-LABEL: and:
+; CHECK:       # %bb.0:
+; CHECK-NEXT:    and a0, a0, a1
+; CHECK-NEXT:    andi a0, a0, 42
```

Comment:
You might want to add an alias from `andi` to `add`, as we've done with ADDI. The instruction is still ANDI - in the ISA and internally to the compiler - but the assemblers both accept the reg/reg form even with an immediate operand. Pretty strange! It should be pretty easy to see how it's done though.

Reply:
Thanks @frasercrmck, done.